src/common/options/crimson.yaml.in

```diff
@@ -45,6 +45,11 @@ options:
   level: advanced
   default: 0
   desc: Report OSD status periodically in seconds, 0 to disable
+- name: crimson_backfill_max_active_per_pg
```

Comment:
"max_active" isn't very descriptive.  This is really max_concurrent_pushes, right?  It seems like this should share a limit with log based recovery -- do we already have a config for that?